homeassistant/components/google_travel_time/config_flow.py

```diff
@@ -169,6 +180,10 @@ class GoogleTravelTimeConfigFlow(ConfigFlow, domain=DOMAIN):
 
     VERSION = 1
 
+    def __init__(self) -> None:
+        """Init Config Flow."""
+        self._entry: ConfigEntry | None = None
```

Comment:
You can put this directly as class attribute and skip adding `__init__`

Reply:
Wouldn't this cause havoc when more than one ConfigFlow for google_travel_time is active at the same time?

Reply:
Right. Skip that 👍